index.html

```diff
@@ -23,33 +23,151 @@
+                  <li>css</li>
+                  <li>javaScript</li>
+              </ul>
+              <div class="action">
+    
+                  <button class="button" type="button">
+                          See Project
+                      </button>
+    
+              </div>
+          </div>
+        </section>
+
+        <section class="wrapper">
+          <div class="card3">
+              <div class="snapshot"><img src="./images/tonic2card.svg" alt="tonic"></div>
+              <div class="block">
+                  <h1 class="tonic2_text">Tonic</h1>
+                  <div class="subfram">
+                      <p class="sub1">CANOPY</p>
+                      <div class="circle"></div>
+                      <p class="sub2">Back End Dev</p>
+                      <div class="circle"></div>
+                      <p class="sub2">2015</p>
+                  </div>
+                  </div>
+                  <div class="normal1">A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+                  <ul class="techs">
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javaScript</li>
+                  </ul>
+                  <div class="action">
+        
+                      <button class="button" type="button">
+                              See Project
+                          </button>
+        
+                  </div>
+              </div>
+            </section>
+        
+                <section class="wrapper">
+                  <div class="card4">
+                      <div class="snapshot"><img src="./images/mutipost2.svg" alt="multipost2"></div>
+                      <div class="block">
+                          <h1 class="multipost_text">Multi-post <br> stories</h1>
+                          <div class="subfram">
+                              <p class="sub1">CANOPY</p>
+                              <div class="circle"></div>
+                              <p class="sub2">Back End Dev</p>
+                              <div class="circle"></div>
+                              <p class="sub2">2015</p>
+                          </div>
+                          </div>
+                          <div class="normal">A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+                          <ul class="techs">
+                              <li>html</li>
+                              <li>css</li>
+                              <li>javaScript</li>
+                          </ul>
+                          <div class="action">
+                
+                              <button class="button" type="button">
+                                      See Project
+                                  </button>
+                
+                          </div>
+                      </div>
+                    </section>                              
 </div>
```

Comment:
- [ ] Kindly add the technologies section, I cant see it in your project, please ensure your fix the issues highlighted by the previous reviewer.

Reply:
That section is in the about myself file